gravitee-am-reporter/gravitee-am-reporter-kafka/src/test/java/io/gravitee/am/reporter/kafka/JUnitConfiguration.java

```diff
@@ -35,31 +40,56 @@
 public class JUnitConfiguration implements ApplicationListener<ContextClosedEvent> {
 
     private KafkaContainer kafkaContainer;
+    private SchemaRegistryContainer schemaRegistryContainer;
+    private static final Network NETWORK = Network.newNetwork();
+    private static final String CONFLUENT_PLATFORM_VERSION = "7.6.1";
 
     @Bean
-    public KafkaContainer kafkaContainer(){
-        this.kafkaContainer = new KafkaContainer(
-            DockerImageName.parse("confluentinc/cp-kafka:6.1.1"));
+    public KafkaContainer kafkaContainer() {
+        kafkaContainer = new KafkaContainer(
+                DockerImageName.parse("confluentinc/cp-kafka:" + CONFLUENT_PLATFORM_VERSION))
+                .withNetwork(NETWORK).withReuse(true);
         kafkaContainer.start();
         return kafkaContainer;
     }
 
-    @Override
-    public void onApplicationEvent(ContextClosedEvent contextClosedEvent) {
-        if(this.kafkaContainer != null){
-            this.kafkaContainer.stop();
-        }
+    @Bean
+    public SchemaRegistryContainer schemaRegistryContainer(KafkaContainer kafkaContainer) {
+        schemaRegistryContainer = new SchemaRegistryContainer(CONFLUENT_PLATFORM_VERSION);
+        schemaRegistryContainer.withNetwork(NETWORK).withKafka(kafkaContainer).withReuse(true).start();
```

Comment:
`withReuse()` isn't needed (and won't work anyway wihout an additrional global properties file) - please remove from both containers